about.html

```diff
@@ -0,0 +1,112 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="css/about.css">
+    <link rel="stylesheet" href="css/styles.css">
+    <title>Men International Conference 2023</title>
+</head>
+<body>
+ <header>
+    <nav>
+        <div class="top-nav">
+            <div class="top-nav__container">
+                <img src="asset/images/facebook.svg" alt="facebook-logo" class="icon">
+            <img src="asset/images/twitter.svg" alt="facebook-logo" class="icon">
+            <ul class="top-nav__menu">
+                <li class="top-nav__menu-item"><a href="#">English</a></li>
+                <li class="top-nav__menu-item"><a href="#">My Page</a></li>
+                <li class="top-nav__menu-item"><a href="#">Logout</a></li>
+            </ul>
+            </div>
+        </div>
+        <div class="lower-nav">
+           <div class="lower-nav__center hide" id="menu">
+            <img src="asset/images/logo.png" alt="logo" class="lower-nav__logo">
```

Comment:
According to the project requirements the logo should link to the home page. Please implement this🙏